providers/github/resources/github.lr

```diff
@@ -344,6 +346,12 @@ private github.repository @defaults("fullName") {
   closedIssues() []github.issue
   // Repository license
   license() github.license
+  // Repository code of conduct
+  codeOfConduct() github.file
```

Comment:
Can we get file in the names so it's more obvious what you're getting back from the field name